### DISCUSSION TITLE
Install Pgroonga with Postgres

### DISCUSSION THREAD

**Discussion by thaomd-2569** [pgroonga/pgroonga#371] [Q&A] [answered]
Im using postgres ver 13.4 and want to install pgroonga ?
So what is next step in docker file. Please help me? This is my docker file

`FROM postgres:13.4

RUN apt-get update && apt-get install -y \
    tar \
    build-essential \
    zlib1g-dev \
    liblzo2-dev \
    libmsgpack-dev \
    libzmq-dev \
    libevent-dev \
    libmecab-dev \
    && rm -r /var/lib/apt/lists/*

ENV PGROONGA_VERSION=3.1.5 \
    GROONGA_VERSION=13.0.8

RUN wget https://packages.groonga.org/source/groonga/groonga-13.0.8.tar.gz
RUN tar xvzf groonga-13.0.8.tar.gz
RUN cd groonga-13.0.8  

COPY ./db.sql /docker-entrypoint-initdb.d/`

**Comment by komainu8** [comment, marked as answer]
We provide docker image in https://github.com/pgroonga/docker.
We can use this docker image as below.

```
$ docker run -d groonga/pgroonga:3.1.5-debian-13
```

How about you use docker image in https://github.com/pgroonga/docker?
Please refer https://github.com/pgroonga/docker#how-to-use also.

---

If you want to install PGroonga into environment that PostgreSQL 13.4 already has been installed, you only install PGroonga's package for Debian.

See: https://pgroonga.github.io/install/debian.html

You don't need to build Groonga from source.

**Reply by thaomd-2569** [reply]
@komainu8 
Thank you for your assistance in solving this problem
